nuget/spec/dependabot/nuget/update_checker/version_finder_spec.rb

```diff
@@ -456,10 +456,10 @@
       let(:dependency_name) { "NuGet.Protocol" }
       let(:dependency_version) { "6.3.0" }
 
-      # skipped
-      # it "returns the expected version" do
-      #   expect(subject[:version]).to eq(version_class.new("6.5.0"))
-      # end
+      it "returns the expected version" do
+        skip "This test was commented out and does not work at the moment"
```

Comment:
Same here, is there are plan to address this, or will this realistically just remain skipped?